File: 2tdsph/aula56/AgendaContatoCompleta/app/src/main/java/edu/curso/agendacontatocompleta/model/Contato.kt
package edu.curso.agendacontatocompleta.model

import java.time.LocalDate
import java.util.Calendar
import java.util.Date

data class Contato(
    var id : String = "",
    var nome : String = "",
    var email : String = "",
    var telefone : String = "",
    var nascimento : Date = Calendar.getInstance().time
)

File: 1tdspm/aula53/Contato.py
class Contato:
    def __init__(self, nome : str = "",
                 telefone : str = "",
                 email : str = ""):
        self.nome = nome
        self.telefone = telefone
        self.email = email

    def dicionario(self) -> dict:
        contato_dicionario = {"nome": self.nome,
                              "telefone": self.telefone,
                              "email": self.email}
        return contato_dicionario

    def __str__(self) -> str :
        texto = f"""Nome: {self.nome}\t
        Telefone:{self.telefone}\t
        Email:{self.email}"""
        return texto
    
# c1 = Contato("Joao Silva", "(11) 1111-1111")
# print("Contato: ", c1)
